platform/server/plugins/types.ts

```diff
@@ -77,6 +77,17 @@ export interface BasePluginsType {
   [key: string]: any;
 }
 
+export type KibanaPlugin<
+  DependenciesType extends BasePluginsType,
+  ExposableType = void
+> = {
+  configPath: Array<string> | undefined;
+  dependencies: Array<string>;
```

Comment:
__question:__ Can we make `dependencies` an optional property so that we don't have to define empty array for "independent" plugins?

Reply:
✅